InnoTopicWebsite/src/app/work-projects/work-projects-data.ts

```diff
@@ -61,10 +61,10 @@ export class WorkOrganisationsAndProjectsData {
     logoFileName: `epic-elite-logo.svg`,
     description: 'Test #Rust #React and #AWS hashtags',
     projects: {
-      'AI-based LLM-powered processing of product metadata': project({
+      'EpicElite Platform': project({
```

Comment:
Hmmm @samyakjainnnnn EpicElite is also present below in line 131 - why it was modified here?